app.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+  end
+
+  def list_all_books
+    @books.each do |book|
+      puts "Title: #{book.title}   Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "[#{person.class.name}] Name: #{person.name} ID: #{person.id} Age: #{person.age}"
+    end
+  end
+
+  def create_person(type)
+    age = prompt_input('Age:').to_i
+    name = prompt_input('Name:')
+    specialization = type.downcase == 'teacher' ? prompt_input('Specialization:') : nil
+    parent_permission = prompt_yes_no('Has parent permission?')
```

Comment:
- [ ] **[OPTIONAL]** You worked on this milestone well! 💯 However, it would be nice to include a condition for the parent permission prompt: Teachers don't need parent permission. 🤓 🔧

Reply:
@anagudelogu  I just added it because it was part of the parameters on class Teacher but sure it doesn't make sense, I am correcting it